extension-manifest-v2/src/utils/api.js

```diff
@@ -33,22 +33,27 @@ class Api {
 			return this._refreshPromise;
 		}
 
-		this._refreshPromise = fetch(`${this.config.AUTH_SERVER}/api/v2/refresh_token`, {
-			method: 'POST',
-			credentials: 'omit',
-		}).then((res) => {
-			if (res.status > 400) {
-				throw res.json();
```

Comment:
this path is no removed. Is this intentional? afair callers are testing this returned value for errors.

Reply:
It is intentional. This is the part where it is being called:
https://github.com/ghostery/ghostery-extension/blob/455a51f087084baa7885557c00513ee2cf1c2568/extension-manifest-v2/src/utils/api.js#L125

```
this.refreshToken()
	.then((res) => {
		const { status } = res;
		if (status >= 400) {
		  ...
		}
		this._sendReq(method, path, body)
		 ...
	..
        .catch(err => reject(err));
```

So, the caller of refreshToken expects that refreshToken does not reject and it will handle the status >= 400 itself.